src/Serilog.Sinks.MSSqlServer/Sinks/MSSqlServer/Platform/SqlConnectionFactory.cs

```diff
@@ -16,6 +17,16 @@ public SqlConnectionFactory(string connectionString, bool useAzureManagedIdentit
                 throw new ArgumentNullException(nameof(connectionString));
             }
 
+            // Add 'Enlist=false', so that ambient transactions (TransactionScope) will not affect logging
+            // unless connectionstring already contains Enlist
+            //  to contain Enlist the word shoudld be at the beginning or after ';'
+            //  and contain a '=' after, with some optional space before or after the word
+            const RegexOptions regexOptions = RegexOptions.IgnoreCase | RegexOptions.CultureInvariant | RegexOptions.ExplicitCapture;
```

Comment:
Please make this behavior configurable. Add an option in the SinkOptions (e.g. ExemptLoggingFromDatabaseTransactionScopes) with default true and do not add "Enlist=false" if the user set this option to false. Describe the option in detail in the README.md including a warning that this should only be changed if the user knows what they are doing.